adapters/repos/db/vector_index.go

```diff
@@ -16,9 +16,9 @@ import "github.com/semi-technologies/weaviate/adapters/repos/db/helpers"
 // VectorIndex is anything that indexes vectors effieciently. For an example
 // look at ./vector/hsnw/index.go
 type VectorIndex interface {
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.